### PR TITLE
Crusades can now end

### PR DESCRIPTION
**Pull request by SingingSpock**
## About The Pull Request
<details>
<summary>
	Gives two ways to end the Crusader protocol
</summary>
<hr>
There is now a litany to end the Crusader protocol once it is no longer needed. It will be mandatory in SOP for certain individuals to use this after the crisis is resolved. Additionally, admins can remove the antag status from a person and the Crusade will end for them. There is not currently a way to do this en masse, but it likely wouldn't be hard to code one.
	
<hr>
</details>

## Changelog
:cl:
add: Crusades can now be ended with a litany or by admins
/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Tested 'n good to go?

**Comment by Trilbyspaceclone** [comment]
This pr dosnt fix them still being crusaders, it just removes it from their implant.

File: code/modules/core_implant/cruciform/rituals/crusader.dm [hunk 1]
```diff
@@ -140,3 +140,12 @@
...
 	return TRUE
+
+/datum/ritual/targeted/cruciform/crusader/end_crusade
+	name = "End Crusade"
+	phrase = "Tempus occidendi, et tempus sanandi; tempus destruendi, et tempus aedificandi." //"A time to kill and a time to heal, a time to tear down and a time to build"
+	desc = "Ends your Crusade, returning you to the state you were in when you entered the colony."
+	power = 5 //This is literally a litany to depower yourself, it shouldn't cost much
+
+/datum/ritual/targeted/cruciform/crusader/end_crusade/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C,list/targets)
+	C.remove_crusader()
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/crusader.dm:151]: this bit needs the antag bit removed

Comment (SingingSpock) [reply code/modules/core_implant/cruciform/rituals/crusader.dm]: Done

**Comment by cdb-is-not-good** [comment]
The equipment realistically should return as well, no? Or does that already happen

**Comment by SingingSpock** [comment]
> The equipment realistically should return as well, no? Or does that already happen

Ideally, yeah, but that's way more complicated code-wise. The primary intent with this is to remove the litanies... including Vector litanies from Scalars (which was the thing that made me actually write this, since Heads are going to be Scalar only and I want them able to give up the Vector litanies once the threat's handled).

**Pull request merged**